db/migrate/20221211065235_copy_shipping_name_to_addresses.rb

```diff
@@ -0,0 +1,14 @@
+class CopyShippingNameToAddresses < ActiveRecord::Migration[6.0]
+  def up
+    Order.find_each do |order|
+      order.shipping_address.update_columns(shipping_name: order.shipping_name)
```

Comment:
Forgot to include the `shipping_name` in the address schema initially, so had to come back with a follow-up migration/transition